docs/user-guide/configuration.md

```diff
@@ -142,6 +142,52 @@ Globally configure how `dates` should be formatted. All [python's `strftime` dir
 
 Globally configure how `time` should be formatted. All [python's `strftime` directives](http://strftime.org) are supported.
 
+### Default tags
+
+Tags can be automatically added for some projects. This is convenient when the
```

Comment:
s/for some/to some/

Instead of "some" I would prefer "chosen", "selected", "specific" or similar.